src/compiler/es-next/index.js

```diff
@@ -87,14 +87,9 @@ export default class ESNextCompiler {
 
     _setupSourceMapsSupport () {
         sourceMapSupport.install({
+            hookRequire:              true,
             handleUncaughtExceptions: false,
-            environment:              'node',
-
-            retrieveSourceMap: filename => {
-                var map = this.sourceMaps[filename];
-
-                return map ? { url: filename, map } : null;
-            }
+            environment:              'node'
```

Comment:
They will obtain automatically, ` retrieveSourceMap:` argument used to be custom implementation of this process instead `source-maps-support` default.